oggm_edu/glacier_collection.py

```diff
@@ -596,11 +598,11 @@ def plot_mass_balance(self):
         elas = []
 
         # Plot annual mass balance for each glacier.
-        for i, glacier in enumerate(self.glaciers):
+        for glacier in self.glaciers:
             ax.plot(
                 glacier.annual_mass_balance,
                 glacier.bed.bed_h,
-                label=f"Glacier {i+1}, " + f"gradient {glacier.mass_balance.gradient}",
+                label=f"Glacier {glacier.id}, " + f"gradient {glacier.mass_balance.gradient}",
```

Comment:
This is going to be quite ugly per default. Is there a way to assign incremental IDs when the GlacierCollection is created?

Reply:
It seems we can assign incremental ids when the glacier is created using a class wide counter. Which is much better than the random ids. See https://github.com/OGGM/oggm-edu/pull/188/commits/a6b98dfb881e3d0e827f71b86a294cc79804fb06.